Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -42,8 +42,19 @@ public function handleException(InterpreterExceptionEvent $event)
      */
     private function resolveInterpreterExceptionMessage(InterpreterException $exception)
     {
-        if ($exception instanceof DateTimeFormatException) {
-            return "Error matching date “{$exception->getValue()}” to format “{$exception->getFormat()}”.";
+        if ($exception instanceof FieldValueException) {
+            $message = "The value “{$exception->getActualValue()}” is invalid.";
```

Comment:
Well at the moment we do not translate anything in this bundle. Translating them means that we have to extract them properly for translation etc. What do you think? 

I think even if we decided to translate them I would definitely do this on a different PR